cluster/gce/manifests/etcd.manifest

```diff
@@ -9,14 +9,25 @@
 "securityContext": {
     "seccompProfile": {
         "type": "RuntimeDefault"
-    }
+    },
+    "runAsUser": {{runAsUser}},
+    "runAsGroup": {{runAsGroup}},
+    "runAsNonRoot": true
```

Comment:
not really needed?

Reply:
Setting it to true will prevent the pod run as user 0 which is root, not sure it's desired or not.

Reply:
Fixed.